lib/p2p/Pool.ts

```diff
@@ -128,8 +128,8 @@ class Pool extends EventEmitter {
     // TODO: send only to peers which accepts the pairId
   }
 
-  public broadcastOrderInvalidation = (orderId: string, pairId: string) => {
-    const orderInvalidationPacket = new OrderInvalidationPacket({ orderId, pairId });
+  public broadcastOrderInvalidation = (id: string, pairId: string) => {
```

Comment:
any reason behind renaming `orderId` to `id`? `orderId` was much clearer and easily understood at first glance 😄

Reply:
I thought that is kind of redundant because there is already 'order' in the name of the function and it makes the callback of `subscribePeerOrders` in `Service.ts` more elegant because I can use `{ cancelled: true, ...order }` instead of assigning the value of `orderId` to `id` and removing the `orderId` property.

I revert the change if you want to though? @itsbalamurali